### DISCUSSION TITLE
Ipcalc

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#2227] [Ideas]
Hi,

As a beginner I was a bit confused by the LOCAL_NETWORK parameter.
The tool ipcalc really helped me know what to put; I did:
sudo apt install ipcalc
ifconfig (found the IP associated with my interface)
then ipcalc <the_ip> and that was I needed to put for <LOCAL_NETWORK>

Just a suggestion but maybe this could be added in the documentation.

**Comment by pkishino** [comment]
Feel free to add this as a tip ;)
